amr-wind/equation_systems/icns/icns_advection.H

```diff
@@ -417,6 +421,16 @@ struct AdvectionOp<ICNS, fvm::Godunov>
             }
         }
 
+        // Multiphase flux operations
+        if (mphase_vof) {
```

Comment:
Is this the part you override the fluxes ?

Reply:
Yes, this is the part where the fluxes are overridden, as well as converted to mass-consistent momentum fluxes.